docs/data/upload.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Upload Data"
+linkTitle: "Upload Data"
+description: "Upload data to the Viam app from your local computer or mobile device using the data client API, Viam CLI, or Viam mobile app."
+weight: 40
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+image: "/services/icons/data-capture.svg"
+imageAlt: "Upload data to the Viam app"
+images: ["/services/icons/data-capture.svg"]
+# SME: Alexa Greenberg
+---
+
+If you configured [data capture](/data/capture/) on your machine, data is automatically uploaded to the Viam cloud from the directory and at the interval you specified.
+However, if you want to upload a batch of data manually from somewhere else either on your machine, or from your personal computer or mobile device, you have several options:
+
+- Run a Python script to upload files from a folder using the data client API `file_upload_from_path` method.
+- Run a Golang script calling the data management service `sync` method.
+- Upload images from your mobile device using the Viam mobile app.
+
+## Upload data with a Python script
+
+You can use the Viam Python SDK's data client API [`file_upload_from_path`](/build/program/apis/data-client/#fileuploadfrompath) method to upload each file you'd like to upload from your computer to the Viam cloud.
+
+1. Install the [Viam Python SDK](https://python.viam.dev/) by running the following command on the computer from which you want to upload data:
+
+   ```sh {class="command-line" data-prompt="$"}
+   pip install viam-sdk
+   ```
+
+2. Create a Python script file in a directory of your choice and [add code to establish a connection](/build/program/apis/data-client/#establish-a-connection) from your computer to your [Viam app](https://app.viam.com) {{< glossary_tooltip term_id="location" text="location" >}} or individual {{< glossary_tooltip term_id="part" text="machine part" >}}.
+
+3. To upload just one file, make a call to the `file_upload_from_path` method according to [the data client API documentation](/build/program/apis/data-client/#fileuploadfrompath).
+   The following example code could be placed inside the `main()` function (or a function called from `main()`):
+
+   ```python {class="line-numbers linkable-line-numbers"}
+   await data_client.file_upload_from_path(
+     # The ID of the machine part the file should be associated with
+     part_id="abcdefg-1234-abcd-5678-987654321xyzabc",
+     # Any tags you want to apply to this file
+     tags=["cat", "animals", "brown"],
+     # Path to the file
+     filepath="/Users/Artoo/my_cat_photos/brown-cat-on-a-couch.png"
+   )
+   ```
```

Comment:
```suggestion
     ```python {class="line-numbers linkable-line-numbers"}
     await data_client.file_upload_from_path(
       # The ID of the machine part the file should be associated with
       part_id="abcdefg-1234-abcd-5678-987654321xyzabc",
       # Any tags you want to apply to this file
       tags=["cat", "animals", "brown"],
       # Path to the file
       filepath="/Users/Artoo/my_cat_photos/brown-cat-on-a-couch.png"
     )
     ```
```